src/Core/Checkout/Cart/Validator.php

```diff
@@ -18,13 +19,16 @@ public function __construct(private readonly iterable $validators)
     {
     }
 
+    /**
+     * @return array<Error>
```

Comment:
```suggestion
     * @return list<Error>
```

array_values returns a list